CaT/src/TableRelations/TableFactory.php

```diff
@@ -129,6 +129,45 @@ public function histUserOrgu($id) {
 	}
 
 	/**
+	 * Hist user orgu table representation
+	 *
+	 * @param	string	$id
+	 * @return	AbstractTable
+	 */
+	public function histUserTestrun($id) {
+		return $this->table('hist_usertestrun',$id)
+			->addField($this->field('usr_id'))
+			->addField($this->field('obj_id'))
+			->addField($this->field('pass'))
+			->addField($this->field('hist_historic'))
+			->addField($this->field('max_points'))
+			->addField($this->field('points_achieved'))
+			->addField($this->field('test_passed'))
+			->addField($this->field('testrun_finished_ts'))
+			->addField($this->field('test_title'))
+			->addField($this->field('percent_to_pass'))
+		;
+	}
+
+	public function allOrgusOfUsers($id,array $usr_ids = array()) {
+
+		$orgus = $this->histUserOrgu('orgus');
+		$constraint = $orgus->field('hist_historic')->EQ()->int(0)->_AND($orgus->field('action')->GE()->int(0));
+		if(count($usr_ids)>0) {
+			$constraint = $constraint->_AND($this->predicate_factory->IN($orgus->field('usr_id'),$this->predicate_factory->list_int_by_array($usr_ids)));
+		}
+		$orgus->addConstraint($constraint);
+		$all_orgus_space = $this->TableSpace()
+					->addTablePrimary($orgus)
+					->setRootTable($orgus)
+					->request($orgus->field('usr_id'))
+					->request($this->groupConcatFieldSql('orgus',$orgus->field('orgu_title')))
+					->groupBy($orgus->field('usr_id'));
+
+		return $this->DerivedTable($all_orgus_space,$id);
+	}
```

Comment:
I can't see how these belong in this general table factory class. Should have said that for the other hist*-tables already. I think this belongs to the Historizing Plugin. How should i ever know, that i need to change these fields here, when i change them in the HistorizingStorage?

Please at least write a ticket for this issue, better refactor before the next release.